### PR TITLE
docs(server-jaeger): add documentation on disabling jaeger tracing

### PR DESCRIPTION
**Pull request by Fox32**

### CODE REVIEW

File: sda-commons-server-jaeger/README.md [hunk 1]
```diff
@@ -65,3 +65,37 @@ To override it with a custom name for each deployment, pass the `JAEGER_SERVICE_
 By default, sampling is configured to sample every trace.
 This is fine for testing or development, but might be to much data for production.
 Therefore specify `JAEGER_SAMPLER_TYPE` and `JAEGER_SAMPLER_PARAM` in your deployments, see the [sampler types](https://www.jaegertracing.io/docs/1.16/sampling/#client-sampling-configuration) for more information.
+
+### Disabling Jaeger Tracing
+
+In you are running your service in an environment where Jaeger is unavailable, you might want to disable the Jaeger tracing.
```
Comment (JoergSiebahn) [inline sda-commons-server-jaeger/README.md:71]:
```suggestion
If you are running your service in an environment where Jaeger is unavailable, you might want to disable the Jaeger tracing.
```

File: sda-commons-server-jaeger/README.md [hunk 2]
```diff
@@ -65,3 +65,37 @@ To override it with a custom name for each deployment, pass the `JAEGER_SERVICE_
...
+	at io.jaegertracing.thrift.internal.senders.UdpSender.send(UdpSender.java:83)
+	... 4 common frames omitted
+Caused by: java.net.PortUnreachableException: ICMP Port Unreachable
+	at java.base/java.net.PlainDatagramSocketImpl.send(Native Method)
+	at java.base/java.net.DatagramSocket.send(Unknown Source)
+	at io.jaegertracing.thrift.internal.reporters.protocols.ThriftUdpTransport.flush(ThriftUdpTransport.java:146)
+	... 9 common frames omitted
+```
+
+To disable tracing you have configure it to sample no spans.
```
Comment (JoergSiebahn) [inline sda-commons-server-jaeger/README.md:100]:
```suggestion
To disable tracing you have to configure it to sample no spans.
```

File: sda-commons-server-jaeger/README.md [hunk 3]
```diff
@@ -65,3 +65,37 @@ To override it with a custom name for each deployment, pass the `JAEGER_SERVICE_
 By default, sampling is configured to sample every trace.
 This is fine for testing or development, but might be to much data for production.
 Therefore specify `JAEGER_SAMPLER_TYPE` and `JAEGER_SAMPLER_PARAM` in your deployments, see the [sampler types](https://www.jaegertracing.io/docs/1.16/sampling/#client-sampling-configuration) for more information.
+
+### Disabling Jaeger Tracing
+
+In you are running your service in an environment where Jaeger is unavailable, you might want to disable the Jaeger tracing.
+While it's not a problems if sampling data can't be reported to the Jaeger backend, you might get the following warning in your logs:
```
Comment (dhenneke) [inline sda-commons-server-jaeger/README.md:72]:
```suggestion
While it's not a problem if sampling data can't be reported to the Jaeger backend, you might get the following warning in your logs:
```

**Comment by Fox32** [comment]
@dhenneke has tested this and can confirm that it works.

**Review by dhenneke — APPROVED** [review]

**Pull request merged**